### PR TITLE
Fetch cobs by selected or default state

### PR DESCRIPTION
**Pull request by sebastinez**
Currently we don't query by state, which means we get patches of different states and by paginating them we eventually loose patches that should be shown by selecting their state.

Created a patch on heartwood for the `radicle-httpd` part with id `2aaaf04d2c1335b0db41f7d401d18de6d29292a6` but isn't being replicated yet on seed.radicle.xyz.

### CODE REVIEW

File: src/views/projects/View.svelte
Comment (rudolfs) [inline src/views/projects/View.svelte:39]: 👀

**Comment by rudolfs** [comment]
LGTM. Let's merge this once the backend is deployed.

**Comment by sebastinez** [comment]
So I finally created a PR since my radicle patch didn't get to seed.radicle.xyz 
https://github.com/radicle-dev/heartwood/pull/534 let's wait for it be merged and deployed

File: src/views/projects/View.svelte
Comment (sebastinez) [reply src/views/projects/View.svelte]: a393d0b

**Review by rudolfs — APPROVED** [review]

**Pull request merged**
